app/models/concerns/created_updated_scopes.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+# Observation.created_at("2005-03-04").to_sql
```

Comment:
What does this comment mean?

Also, is there some way to let readers know where these scopes will be applied?  "created_updated_scopes" is fairly opaque to me.  I mean, sure I can search for where this is included and find out without too much difficulty, but maybe a better file name and/or some comments could save the trouble?  I assume this must have something to do with observations based on the cryptic comment?

Reply:
Oh!!!!  I see these scopes apply to all models with created_at and updated_at.  I'm slow.  Hmmm.  Is there a better name?...

"concern" is opaque to me, personally.  But I never liked jargony abstract terms like "factory", "concern", etc.  That may just be a personal failing.  I just wonder if something emphasizing that the files inside are shared between models might work better.  Is that what "concern" signals normal readers?  Am I being silly to suggest something more straightforward like "shared"?  It works just fine for views, why not models, too?  But maybe "shared" means something else more specific in connection with model architecture?

"created updated scopes" does not work for me, though, because my first instinct is that these are scopes that have been created and then updated.  But in fact no, they are scopes involving created_at and updated_at.  Well, there, wordy as hell, but maybe "scopes_involving_created_at_and_updated_at" would certainly be less ambiguous.  Personally, I would understand what "timestamp_scopes" means, but maybe it wouldn't work for other readers.

Reply:
LOL - I'm much later to the Rails game and still haven't gotten over the feeling that "model", "view" and "controller" are jargony! Although they're not unique to Rails. "Concern" is an extensible class of core Rails though (since about version 3!) and concerns have a few benefits over shared modules that are described cursorily in the [docs](https://api.rubyonrails.org/v6.1.6/classes/ActiveSupport/Concern.html) and a bit better [here](https://dev.to/software_writer/how-rails-concerns-work-and-how-to-use-them-gi6). I vote to keep it a concern. 

I hope you always feel free to suggest renamings! I labor over them but often don't feel like I've nailed them. Happy to change to ScopesInvolvingCreatedAtAndUpdatedAt — TimestampScopes, or ScopesForTimestamps maybe? I had considered DateTimeScopes also. (It has the benefit of sounding like an astrology advice column!)

Reply:
Oh the cryptic comment came from me testing the scopes in the console, and not wanting to keep retyping that! To be deleted.